src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -101,6 +105,22 @@ def create_subvolume(self, spec, size=None, namespace_isolated=True, mode=0o755,
                 xattr_val = self._get_ancestor_xattr(subvolpath, "ceph.dir.layout.pool")
             # TODO: handle error...
             self.fs.setxattr(subvolpath, xattr_key, xattr_val.encode('utf-8'), 0)
+
+            if uid is None:
```

Comment:
What about usingn the uid/gid of the subvoluem group by default?

Reply:
@batrick Fixed for subvolume inherits it's group's uid and gid. 
What do you suggest for subvolume group? I'm leaving the subvolume group defaults as `0`, as `/volumes` is a prefix. But then it is also a directory.

Reply:
Default 0 is probably necessary because CSI and probably Manila rely on that. @ajarr ?

Reply:
The libcephfs client create the directories with uid and gid 0 for CSI, https://github.com/ceph/ceph/commit/033d9e743a9#diff-cfd3b6f517caccc18f7f066395e8a4bdR73 , and, likewise in libcephfs client in ceph_volume_client library for manila.

Reply:
Thanks @ajarr. 
I understand the current setting is good for mgr/volumes:
1. Subvolume inherits it's group's uid and gid by default.
2. Subvolume Group's uid and gid defaults to 0.

If these are okay, please proceed with your teuthology run. Thanks.